src/main/java/org/jabref/logic/formatter/bibtexfields/NormalizeNamesFormatter.java

```diff
@@ -24,6 +24,7 @@ public String getKey() {
     @Override
     public String format(String nameList) {
         Objects.requireNonNull(nameList);
+        nameList = nameList.replaceAll(" and", ",");
```

Comment:
Please don't change the `format` method but only `AuthorList.parse` since the former has almost no test coverage while for the latter exists an extensive test suite.

Reply:
done :)

Reply:
Thanks! Did you also run the tests in `AuthorListTests`. I'm pretty sure that some of them now fail. The problem is that now `A, B,C and D, E` and `A, B and C, D, E` are mapped to the same `A, B, C, D, E` and the parser can no longer decide between those two inequivalent inputs.